content/en/blog/2022/gc-election-results.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Announcing the 2022 Governance Committee Election Results
+linkTitle: 22 GC Election Results
+date: 2022-10-21
+author: Austin Parker
+---
+
+The OpenTelemetry project is proud to announce the winners of the 2022
+OpenTelemetry Governance Committee (GC) election! Before we get into the
+results, I'd like to share some statistics about the election!
+
+This year, 513 project members were eligible to vote. Of those, 219 cast
+ballots, for a participation rate of nearly 43%! For comparison, the 2021
+election saw 169 ballots cast with 386 eligible voters. Participation was 
+roughly flat year-over-year, and we're thrilled that we've welcomed so 
+many new project members and maintained a high participation rate.
+
+With that out of the way, please joining me in congratulating [Alolita
```

Comment:
Can you confirm you did mean "joining" here? As a non-native speaker, I would write "join me in" instead, but I'm probably wrong.

Reply:
I did not! Good catch